compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
There is the `pull` command docs file that'll need updating with the new option as well. I think that's it but I'll double check now.

https://github.com/docker/compose/blob/master/docs/reference/pull.md

Thank you for your work on this, it's appreciated.